src/lib/operators/join_sort_merge.cpp

```diff
@@ -39,19 +46,13 @@ JoinSortMerge::JoinSortMerge(const std::shared_ptr<const AbstractOperator>& left
                              const OperatorJoinPredicate& primary_predicate,
                              const std::vector<OperatorJoinPredicate>& secondary_predicates)
     : AbstractJoinOperator(OperatorType::JoinSortMerge, left, right, mode, primary_predicate, secondary_predicates) {
-  // Validate the parameters
-  Assert(mode != JoinMode::Cross, "This operator does not support cross joins.");
-  Assert(left, "The left input operator is null.");
-  Assert(right, "The right input operator is null.");
   Assert(primary_predicate.predicate_condition == PredicateCondition::Equals ||
```

Comment:
Your reason to keep that assert?

Reply:
I moved it to AbstractJoinOperator's constructor. We should have some place making sure JoinOps are not called with IN/EXISTS predicates